components/kyma-environment-broker/cmd/cli/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/cmd/cli/logger"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/cmd/cli/skr"
+)
+
+func main() {
+	setupCloseHandler()
+	logger := logger.New()
```

Comment:
field's name collides with the imported package

Reply:
Changed